contracts/libs/Types.sol

```diff
@@ -29,9 +29,8 @@ library Types {
         bytes32 depositTree;
         address committer;
         bytes32 txRoot;
-        uint256 stakeCommitted;
         uint256 finalisesOn;
-        uint256 timestamp;
+        bool withdrawn;
```

Comment:
It looks like `stakeCommitted` also acts as the state of the batch being withdrawn or slashed, thus adding a withdrawn field to replace that.
The state of `slashed` is a little bit strange, see next comments